### PR TITLE
iMX8MQ SPL/FIT secure boot

### PR DESCRIPTION
**Pull request by igoropaniuk**
Please do not submit a Pull Request via github.  Our project makes use of
mailing lists for patch submission and review.  For more details please
see https://www.denx.de/wiki/U-Boot/Patches

### CODE REVIEW

**Comment by ricardosalveti** [comment]
Why do we need to drop the hardcoded USB configuration? Just not clear from the commit log.

**Comment by igoropaniuk** [comment]
> Why do we need to drop the hardcoded USB configuration? Just not clear from the commit log.

@ricardosalveti added explanation to the commit message

File: include/configs/imx8mq_evk.h [hunk 1]
```diff
@@ -31,7 +31,6 @@
 /* For RAW image gives a error info not panic */
 #define CONFIG_SPL_ABORT_ON_RAW_IMAGE
 
-#undef CONFIG_DM_MMC
```
Comment (mike-scott) [inline include/configs/imx8mq_evk.h:34]: Commit title is incorrect here:
`[FIO internal] imx8mq_evk: disable DM_I2C for SPL build`

**Review by mike-scott — CHANGES_REQUESTED** [review]

File: include/configs/imx8mq_evk.h (hunk 1 above)
Comment (igoropaniuk) [reply include/configs/imx8mq_evk.h:34]: ah sorry, fixed and re-pushed

**Review by mike-scott — APPROVED** [review]
LGTM

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**